src/common/models/Projection.ts

```diff
@@ -379,7 +392,9 @@ export class Projection {
 
     // only keep futureDaysToInclude days ahead of today
     const now = new Date();
-    const days = dates.findIndex(date => date > now) + futureDaysToInclude;
+    const todayIndex = dates.findIndex(date => date > now);
+    const days =
+      todayIndex >= 0 ? todayIndex + futureDaysToInclude : dates.length;
```

Comment:
The old code didn't work right in the case that there are no future days (which is the norm if there are no projections)